ztag/schema.py

```diff
@@ -400,6 +400,28 @@
     "metadata":local_metadata
 })
 
+ztag_fox = SubRecord({
+    "version": AnalyzedString(es_include_raw=True),
+    "id": Integer(),
+    "hostname": AnalyzedString(es_include_raw=True)
+    "host_address": AnalyzedString(es_include_raw=True),
+    "app_name": AnalyzedString(es_include_raw=True),
+    "app_version": AnalyzedString(es_include_raw=True),
```

Comment:
It doesn't, but I'll fix. I essentially decided there are so few, let's just make all analyzed.